rpc/src/main/kotlin/dk/cachet/carp/rpc/GenerateExampleRequests.kt

```diff
@@ -187,6 +187,7 @@ private val bikeBeaconPreregistration = bikeBeacon.createRegistration {
     organizationId = UUID( "4e990957-0838-414c-bf25-2d391e2990b5" )
     majorId = 42
     minorId = 42
+    additionalSpecifications = "model: 1234"
```

Comment:
Similar to `applicationData = "{\"uiTheme\": \"black\"}"` and other "application data" fields for which examples are created here, let's make this a representative JSON string.